packages/web/src/index.css

```diff
@@ -392,3 +392,18 @@
     'Liberation Mono', 'Courier New', 'monospace';
   font-size: 14px;
 }
+
+.cm-tooltip {
```

Comment:
I think adding a box shadow here would be nice, particularly in light mode to give some depth.

Maybe `shadow-md` from tailwind?

```css
  box-shadow:
    0 4px 6px -1px rgb(0 0 0 / 0.1),
    0 2px 4px -2px rgb(0 0 0 / 0.1);
```